com.unity.ml-agents.extensions/Runtime/Sensors/GridSensor.cs

```diff
@@ -98,6 +99,18 @@ public enum GridDepthType { Channel, ChannelHot };
         [Tooltip("The reference of the root of the agent. This is used to disambiguate objects with the same tag as the agent. Defaults to current GameObject")]
         public GameObject rootReference;
 
+        const int k_AbsoluteMaxCollidersPerCell = 500;
```

Comment:
I'd prefer to have this a parameter too - if someone wants to blow up their memory in order to get maximum fidelity, we should at allow them the option of doing it.

Reply:
ok, was hoping you'd comment on it.